sdk-core/src/main/java/co/reachfive/identity/sdk/core/PasswordClient.kt

```diff
@@ -76,16 +76,16 @@ internal class PasswordAuthClient(
                     success = {
                         if(it.mfaRequired == true) {
                             if(mfaConf == null) {
-                                failure(ReachFiveError.from("A redirect url and an activity are required to achieve a login with password."))
+                                failure(ReachFiveError.from("An activity is required to achieve a login with password."))
```

Comment:
Default `LoginMfaConf.redirectUri` to null in the constructor, and mark the field as deprecated?